distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/admin-app.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "admin-app",
```

Comment:
❓ Do all of these really need `-app` in their names?  The same goes for the file name itself.  It all seems redundant since it is already in an `applications` folder.

Reply:
This names must remain identical or else the various plugins can't be associated with the app. I can change the file names though.